spec/acceptance/resource_cmd_spec.rb

```diff
@@ -6,15 +6,19 @@
 describe 'puppet resource firewall command' do
   before(:all) do
     # In order to properly check stderr for anomalies we need to fix the deprecation warnings from puppet.conf.
-    config = shell('puppet config print config').stdout
-    shell("sed -i -e \'s/^templatedir.*$//\' #{config}")
+    config = run_shell('puppet config print config').stdout
+    run_shell("sed -i -e \'s/^templatedir.*$//\' #{config}")
+    run_shell('echo export LC_ALL=C > ~/.bashrc')
```

Comment:
should this live somewhere else ? other modules likely need this

Reply:
this was needed for el platforms provisioned with docker